content/patterns/checkbox/examples/checkbox-mixed.html

```diff
@@ -18,103 +18,104 @@
+          <li>支援技術の利用者が、チェックボックスがすべて「サンドイッチの調味料」という名前の関連チェックボックスのグループの一部であることを理解できるように、チェックボックスは <code>legend</code> 要素でラベル付けされた <code>fieldset</code> 要素でラップされています。</li>
+          <li>ラベル又はチェックボックスのいずれかをクリックするとチェックボックスがアクティブになることを知覚しやすくするために、ポインターがチェックボックス又はラベルのいずれかにホバーすると、背景色が変わり、境界線が表示され、カーソルがポインターに変わります。</li>
           <li>
-            Because transparent borders are visible on some systems when operating system high contrast settings are enabled, transparency cannot be used to create a visual difference between the element that is focused and other elements.
-            Instead of using transparency, the focused element has a thicker border and less padding.
-            When an element receives focus, its border changes from 0 to 2 pixels and padding is reduced by 2 pixels.
-            When an element loses focus, its border changes from 2 pixels to 0 and padding is increased by 2 pixels.
+            オペレーティングシステムのハイコントラスト設定が有効になっている場合、一部のシステムでは透明な境界線が表示されるため、フォーカスされている要素と他の要素との間に視覚的な違いを作成するために透明度を使用することはできません。
+            透明度を使用する代わりに、フォーカスされた要素はより太い境界線とより少ないパディングを持ちます。
+            要素がフォーカスを受け取ると、その境界線は0から2ピクセルに変わり、パディングは2ピクセル減少します。
+            要素がフォーカスを失うと、その境界線は2ピクセルから0に変わり、パディングは2ピクセル増加します。
           </li>
           <li>
-            To ensure the borders of the inline SVG checkbox graphics in the CSS have sufficient contrast with the background when high contrast settings invert colors, the color of the borders are synchronized with the color of the text content.
-            For example, the color of the checkbox borders is set to match the foreground color of high contrast mode text by specifying the CSS <code>currentcolor</code> value for the <code>stroke</code> property of the <code>rect</code> and <code>polyline</code> elements used to draw the checkbox.
-            To make the background of the checkbox graphics match the high contrast background color, the <code>fill-opacity</code> attribute of the <code>rect</code> element is set to zero.
-            If specific colors were instead used to specify the <code>stroke</code> and <code>fill</code> properties, those colors would remain the same in high contrast mode, which could lead to insufficient contrast between the checkbox and the background or even make the checkbox invisible if the color matched the high contrast mode background.
-            <br>Note: The SVG element needs to have the CSS <code>forced-color-adjust</code> property set to <code>auto</code> for the <code>currentcolor</code> value to be updated in high contrast mode.
-            Some browsers do not use <code>auto</code> for the default value.
+            ハイコントラスト設定で色が反転した場合に、CSS のインライン SVG チェックボックスグラフィックの境界線が背景と十分なコントラストを持つようにするために、境界線の色はテキストコンテンツの色と同期されます。
```

Comment:
ensureを訳出するとこんな感じでしょうか
```suggestion
            ハイコントラスト設定で色が反転した場合に、CSS のインライン SVG チェックボックスグラフィックの境界線が背景と十分なコントラストを持つことを保証するために、境界線の色はテキストコンテンツの色と同期されます。
```